lambdas/services/document_deletion_service.py

```diff
@@ -42,8 +44,12 @@ def delete_specific_doc_type(
         doc_type: Literal[SupportedDocumentTypes.ARF, SupportedDocumentTypes.LG],
     ) -> list[DocumentReference]:
         try:
+            filter_builder = DynamoQueryFilterBuilder()
+
+            filter_builder.add_condition("Deleted", AttributeOperator.EQUAL, "")
```

Comment:
The filter builder allows method chaining so can use this here: 
`filter_expression =  filter_builder.add_condition("Deleted", AttributeOperator.EQUAL, "").build()`

Reply:
Done